standard/classes.md

```diff
@@ -2852,6 +2852,32 @@ public class ReadOnlyPoint {
 }
 ```
 
+Although the backing field is hidden, that field may have field-targeted attributes applied directly to it via the automatically implemented property's *property_declaration* ([§15.7.1](classes.md#1571-general)).
+
+> *Example*: The following code
+> ```csharp
+> public class Foo
+> {
+>     [field: NonSerialized]
+>     public string MySecret { get; set; }
+> }
+> ```
+> results in the field-targeted attribute `NonSerialized` being applied to the compiler-generated backing field, as if the code had been written as follows:
+> ```csharp
+> [Serializable]
+> public class Foo
```

Comment:
The earlier definition of class Foo does not have `[Serializable]`. This inconsistency might give the impression that `[field: NonSerialized]` on a property causes `[Serializable]` to be generated on the containing type.

Reply:
I've changed it to use `[ThreadStatic]` - does that work for you? (There aren't many common field-targeted attributes available as examples...)

Reply:
Whoops, yes, I'd need to make it static as well. And at that point, maybe it's adding more complexity. Okay, will go back to NonSerialized.

Reply:
Done. (Making this a much smaller extra commit, of course.)